pages/api/user/index.ts

```diff
@@ -0,0 +1,43 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import prisma from "@prisma";
+
+// TODO: Type the response data
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    switch (req.method) {
```

Comment:
Let's discuss how we want to handle this. Should we have different files for each type of method or should we have all of them in `index.ts`?